pytest/lib/cargo_fuzz.py

```diff
@@ -1,9 +1,26 @@
+# Running `cargo-fuzz` based fuzzers.
 import os
-import subprocess
 import sys
+import subprocess
+
+
+def run(dir: str, fuzz_target: str) -> int:
+    args = ('cargo', 'fuzz', 'run', fuzz_target, '--', '-len_control=0'
+            '-prefer_small=0', '-max_len=4000000', '-rss_limit_mb=10240')
+    os.environ['RUSTC_BOOTSTRAP'] = '1'
```

Comment:
Hmm, why it's needed?

Reply:
To spell this implicitly, but butting fuzzing into our CI we make it our responsibility to deal with any breakage related to compiler version upgrades. This in contrast to all over Rust functionality we use -- if something stable&documented breaks with an upgrade, we can and should complain upstream about that.